python/mdvtools/project_router.py

```
@@ -1,11 +1,15 @@
-import os
-from flask import Flask, Response, request, jsonify, session, redirect, url_for
+import logging
+from flask import Flask, current_app, Response, request, jsonify, session
```

Ruff (0.8.2)

`flask.current_app` imported but unused

Remove unused import: `flask.current_app`

(F401)